### PR TITLE
Hacktoberfest: Added solution for Project Euler problem 83

### PR DESCRIPTION
**Pull request by Stefan-Radu**
### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: project_euler/problem_083/sol1.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+each cell is a vertex connected to its neighbours.
+
+The solution is based on the Bellman-Ford Algorithm
+
+Reference: https://en.wikipedia.org/wiki/Bellman%E2%80%93Ford_algorithm
+
+TIME 0.22s
+
+"""
+
```
Comment (xcodz-dot) [inline project_euler/problem_083/sol1.py:37]:
```suggestion
The solution is based on the Bellman-Ford Algorithm

Reference: https://en.wikipedia.org/wiki/Bellman%E2%80%93Ford_algorithm
"""

```

TIME is not needed here, Machines are both low end and high end. Performance differs on different devices.

Comment (Stefan-Radu) [reply project_euler/problem_083/sol1.py:37]: Okay, understood. I'll also have to modify other PR's. Thanks

Comment (xcodz-dot) [reply project_euler/problem_083/sol1.py]: Thanks

**Comment by xcodz-dot** [comment]
This is now ready

File: project_euler/problem_083/sol1.py (hunk 1 above)
Comment (poyea) [reply project_euler/problem_083/sol1.py]: True as we'll time it in our Travis.

**Comment by xcodz-dot** [comment]
ping @Stefan-Radu

**Comment by xcodz-dot** [comment]
@dhruvmanila check this out

**Comment by dhruvmanila** [comment]
This PR cannot be merged as there a few changes required but the user has deleted the forked repository from where the pull request originated from. Thus, if @Stefan-Radu wants to submit this solution, please open a new pull request for it. Thank you for your cooperation.

**Comment by xcodz-dot** [comment]
@dhruvmanila shall we close it then?

**Pull request closed without merging**